docs/source/features/mixed_precision.rst

```diff
@@ -25,7 +25,7 @@ Overview
 NVIDIA H100 GPU introduced support for a new datatype, FP8 (8-bit floating point), enabling higher throughput of matrix multiplies and convolutions. NeMo uses the NVIDIA `TransformerEngine <https://github.com/NVIDIA/TransformerEngine>`_ (TE) in order to leverage speedups from FP8. The following table summarizes the FP8 related arguments that can be configured in NeMo (`example config setting <https://github.com/NVIDIA/NeMo/blob/2e1814c9f031ad2aeeebad44597365e97253d2c4/examples/nlp/language_modeling/conf/megatron_gpt_config.yaml/#L192-L200>`_). For a more detailed overview, refer to the TE `documentation <https://docs.nvidia.com/deeplearning/transformer-engine/user-guide/index.html>`_, specifically the FP8 `format <https://docs.nvidia.com/deeplearning/transformer-engine/user-guide/api/common.html#transformer_engine.common.recipe.Format>`_ and `recipe <https://docs.nvidia.com/deeplearning/transformer-engine/user-guide/api/common.html#transformer_engine.common.recipe.DelayedScaling>`_.
 
```

Comment:
I am providing line-by-line copyedits to this file here because most of the file is read only.

Line 6. change to NeMo Framework, delete "Further details will be provided shortly," and revise as follows:

Mixed precision training significantly enhances computational efficiency by conducting operations in low-precision format, while selectively maintaining minimal data in single-precision to preserve critical information throughout key areas of the network. NeMo Framework now supports FP16, BF16, and FP8 via Transformer Engine (TE) across most models. 

Line 12. revise.
NeMo Framework supports half-precision FP16 and BF16 computation training via Megatron Core and the distributed optimizer.

Line 13. revise.
This training recipe uses half-precision for all layer computations while maintaining model states (optimizer states and master parameters) in single-precision.

Line 14. fix punctuation.
To avoid repeated data type casting at each layer computation, Megatron Core keeps a separate copy of half-precision parameters that is updated after each optimizer step.

Lines 22-23. delete the subheading "Overview". It is not needed.

Line 25. revise.
NVIDIA H100 GPU introduced support for a new datatype, FP8 (8-bit floating point), enabling higher throughput of matrix multiplies and convolutions. NeMo Framework uses the NVIDIA Transformer Engine to leverage speedups from FP8. The following table summarizes the FP8 related arguments that can be configured in NeMo (example config setting). For a more detailed overview, refer to the Transformer Engine documentation, specifically the FP8 format and recipe.

Line 36. fix capitalization and grammar.
Enables FP8 training. For transformer networks, the QKV, projection, FC1, and FC2 matrix multiplications are executed using the fourth-generation NVIDIA H100 Tensor Cores with FP8 support.

Line 50. fix punctuation and grammar.
Indicates whether to store module-level parameters in FP8. Enabling this option can reduce memory consumption by eliminating the need to store a copy of weights in higher precision (greater than half) when these weights are externally maintained, such as master parameters in the optimizer. For more information, refer to the fp8_model_init API in TE.

Line 55. Change to "Transformer Engine documentation"

Line 57. change to NeMo Framework and fix punctuation.
Performance optimizations that are natively supported in NeMo Framework by enabling FP8 training with TE.

Line 58. Change to "Transformer Engine installation"